optimum/habana/diffusers/schedulers/scheduling_ddim.py

```diff
@@ -179,7 +179,7 @@ def roll_params(self):
     #     return sample
 
     def _get_variance(self, alpha_prod_t, alpha_prod_t_prev):
-        beta_prod_t = 1 - alpha_prod_t
+        beta_prod_t = 1 - alpha_prod_t + 1e-8  # For numerical stability
```

Comment:
I'm not sure, I will investigate this with other pipelines.

Reply:
I tested the DDIM scheduler with other models, it seems like it is only a problem with this model. I am uncertain of the root cause, but this is a sufficient patch in terms of output quality.

Reply:
I had to implement something similar in #1196 along with a patch to eliminate a bug with the scheduler that would cause unexpected `-inf` variance when `timestep = 1`.